### PR TITLE
FRMW-393: Refactor rxEnvironment.Environment to Quark Service

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-393

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxEnvironment.Environment` to Quark Service.

### CODE REVIEW

**Comment by halkon** [comment]
@CITguy the: `.service('Environment'` Service is another one prime for moving over to `svcs`, @parlarjb thoughts?

**Comment by parlarjb** [comment]
@halkon You mean move it to encore-ui-svcs?

**Comment by halkon** [comment]
yes

**Comment by 100stacks** [comment]
## `Environment` Quark Service

### Demo Page
screen shot 2016-01-04 at 2 02 51 pm

### API Page
screen shot 2016-01-04 at 2 03 13 pm

**Comment by parlarjb** [comment]
I don't know... Somehow, because it interacts with `$location`, I feel better about it being in the framework.

Of course, it's also VERY Rackspace-specific, so I could be convinced otherwise.

**Comment by halkon** [comment]
I do believe we have services already in that project that use `$location` however you make a good point.   

It was the business logic that drove my opinion

**Comment by glynnis** [comment]
Design LGTM

File: src/quarks/Environment/docs/examples/Environment.simple.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div ng-controller="EnvironmentSimpleCtrl">
+    <ul>
```
Comment (CITguy) [inline src/quarks/Environment/docs/examples/Environment.simple.html]: Please correct indentation to 2 spaces per indent.

**Comment by CITguy** [comment]
@halkon: I would say that this may qualify as a candidate for migration to SVCS. We can revisit once Quarks are migrated.

**Comment by halkon** [comment]
:+1: just seemed like a candidate, are you marking candidates in the docs, or perhaps better would be to go through all quarks and figure out which to move?

**Comment by 100stacks** [comment]
@CITguy rebased :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**